src/shared/components/PersonCard/styles.scss

```diff
@@ -0,0 +1,81 @@
+.person-card {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+  border: 5px solid gray;
+  box-shadow: 2px 2px 1px 8px lightgray;
+  width: 300px;
+  min-height: 300px;
+  margin-right: 30px;
+}
+
+.person-card__name {
+  display: inline-block;
+  justify-content: space-between;
+  text-align: center;
+  font-size: 2rem;
+  height: 60px;
+  padding: 10px;
+  border-bottom:2px solid gray ;
+  text-align: start;
+  letter-spacing: 2px;
+}
+
+.person-card__gifts {
+  list-style: none;
+  display: block;
+  flex-direction: column;
+  min-height: 300px;
+  min-width: 200px;
+}
+
+.person-card__gift {
+  display: block;
+  border-bottom: 1px solid black;
+  color:gray;
+  justify-content: flex-start;
+  font-size: 1.2rem;
+  height: 70px;
+}
+
+.person-card__gift-name {
+  margin-left: 10px;
+  font-size: 1.4rem;
+}
+
+.person-card__edit-gift{
+  margin-left: 5px;
+}
+
+.person-card__button-delete-gift {
```

Comment:
You use `btn` in some places and in the other `button`. Stick with one convention

Reply:
Fixed!